packages/@okta/vuepress-site/docs/reference/api/push-providers/index.md

```diff
@@ -16,7 +16,7 @@ The Push Providers API supports the following Authorization Schemes:
 * SSWS - [API tokens](/docs/reference/core-okta-api/#authentication)
```

Comment:
@vanngo-okta @jane-kuang-okta should we completely remove this file from the docs?